src/identity/identity-wallet.ts

```diff
@@ -535,6 +588,61 @@ export class IdentityWallet implements IIdentityWallet {
 
     credential.proof = [mtpProof];
 
+    return credential;
+  }
+
+  /**
+   * {@inheritDoc IIdentityWallet.createIdentity}
+   */
+  async createIdentity(
+    opts: IdentityCreationOptions
+  ): Promise<{ did: DID; credential: W3CCredential }> {
+    opts.method = opts.method ?? DidMethod.Iden3;
+    opts.blockchain = opts.blockchain ?? Blockchain.Polygon;
+    opts.networkId = opts.networkId ?? NetworkId.Amoy;
+
+    const tmpIdentifier = opts.seed ? uuid.v5(Hex.encode(sha256(opts.seed)), uuid.NIL) : uuid.v4();
+    opts.seed = opts.seed ?? getRandomBytes(32);
+
+    await this._storage.mt.createIdentityMerkleTrees(tmpIdentifier);
+
+    const revNonce = opts.revocationOpts.nonce ?? 0;
+
+    const { authClaim, pubKey } = await this.createAuthCoreClaim(revNonce, opts.seed);
+
+    await this._storage.mt.addToMerkleTree(
+      tmpIdentifier,
+      MerkleTreeType.Claims,
+      authClaim.hiHv().hi,
+      authClaim.hiHv().hv
+    );
+
+    const claimsTree = await this._storage.mt.getMerkleTreeByIdentifierAndType(
+      tmpIdentifier,
+      MerkleTreeType.Claims
+    );
+
+    const currentState = hashElems([
+      (await claimsTree.root()).bigInt(),
+      ZERO_HASH.bigInt(),
+      ZERO_HASH.bigInt()
+    ]);
+
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const didType = buildDIDType(opts.method!, opts.blockchain!, opts.networkId!);
```

Comment:
I would suggest here to call it like this: 
```
buildDIDType(opts.method || DidMethod.Iden3, opts.blockchain || Blockchain.Polygon , opts.networkId || NetworkId.Amoy);

```
Then we can remove 3 lines at the top of the function AND remove `eslint-disable-next-line`